IconDiffBot/Core/Application.cs

```diff
@@ -126,7 +131,10 @@ public void Configure(IApplicationBuilder applicationBuilder, IHostingEnvironmen
 					Authorization = new List<IDashboardAuthorizationFilter> { }
 				});
 
-			applicationBuilder.UseMvc();
+			applicationBuilder.UseRouting();
+			applicationBuilder.UseEndpoints(endpoints => {
+				endpoints.MapControllers();
+			});
```

Comment:
I don't know if you tested this, but I was under the impressing that you needed to disable endpoint routing.

If it works it works though

Reply:
This is the only way I *could* make it work